### DISCUSSION TITLE
Did a recent update globally set Sticky Scroll?

### DISCUSSION THREAD

**Discussion by andrewroazen** [alefragnani/vscode-bookmarks#713] [Q&A] [answered]
A few weeks ago I noticed vertical splitting of large PHP/HTML/JS blocks across multiple VS Code projects on different servers that I didn't recall setting; after looking I discovered it was `Sticky Scroll` being checked which I don't remember enabling. Was this a defaults change or did I just enable it and forget doing so? And if it was a change, maybe this should have informed us or asked us before applying it globally? Apologies in advance if this was actually a PEBCAK situation.

**Comment by alefragnani** [comment, marked as answer]
Hi @andrewroazen ,

Do you mean this Sticky Scroll ?

![](https://code.visualstudio.com/assets/docs/getstarted/userinterface/sticky-scroll.png)

If that's the case, yes, the VS Code team has enabled it by default in February release, as documented in their Release notes
https://code.visualstudio.com/updates/v1_87#_editor-sticky-scroll

I guess you opened the issue here, in the Bookmarks extension, because it uses the sticky _terminology_ for its behavior, but both features has no connection.

Hope this helps
